### PR TITLE
Jupyterbook

### PR DESCRIPTION
**Pull request by lelemmen**
This PR will change our website from Docusaurus to Jupyterbook. Jupyterbook allows for using Jupyter notebooks in web pages. Check it out [here](https://jupyterbook.org/intro.html).

### CODE REVIEW

**Comment by lelemmen** [comment]
@dariatols @xdvriend I've set up new website, using Jupyter Book. The old Docusaurus site was fully converted to the new format. The main advantage that Jupyter Book has is that it allows for live code cells and it has automatic links for Google Colab and Binder.

This is just a first version of the new website, I'll be filling up our product backlog if you've approved this PR.

I've done some modifications regarding #808 as well.

File: website/examples/UHF-calculations.ipynb [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "error",
+     "ename": "TypeError",
+     "evalue": "'method' object is not subscriptable",
```
Comment (dariatols) [inline website/examples/UHF-calculations.ipynb:168]: This notebook generates an error. Is this a known issue, or did something go wrong here?

File: website/_toc.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - file: user-documentation/quantum-chemical-methods
+    - file: user-documentation/quantum-chemical-models
+    - file: user-documentation/use-cases
+
+- part: "Examples"
+  chapters:
+    - file: examples/AP1roG-calculations
+    - file: examples/CCSD
+    - file: examples/Custom-algorithms-showcase-CUHF
+    - file: examples/DOCI
```
Comment (dariatols) [inline website/_toc.yml:18]: Including this `DOCI` notebook results in a confusing table of contents:

Screenshot 2020-12-09 at 10 12 22

Comment (dariatols) [reply website/_toc.yml:18]: "Preparing the Hamiltonian in the canonical RHF basis", "Dense DOCI calculations" and "Davidson DOCI calculations" are all sub parts of DOCI calculations and should not be present in the TOC in my opinion.

File: website/examples/README.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
 <img src="../../media/examples.png" width="400">
 </p>
 
+# Examples
```
Comment (dariatols) [inline website/examples/README.md:5]: Is it possible to include all available notebooks into this tab (and not only `CCSD` and `DOCI`) and maybe move it to a separate folder?

**Review by dariatols — APPROVED** [review]
Overall, I like how these notebooks are transformed to webpages. For me, this PR is good to go after addressing some (minor) comments and after the GA build succeeds since it seems to result in an error.

File: website/_toc.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    - file: examples/FCI
+    - file: examples/HF-Stability
+    - file: examples/Hubbard
+    - file: examples/Mulliken-population
+    - file: examples/ONVPath
```
Comment (dariatols) [inline website/_toc.yml:23]: Text / image lay-out in this notebook looks strange.

**Review by xdvriend — APPROVED** [review]
Other than what Daria already said, I have no remarks regarding the implementation, so on that front this PR is good to go!

To be completely honest, I liked the way the old website looked & felt more. I do get that the additional functionality with the notebooks is something incredible, so I get the change. But I hope it will be possible to get the sleek design and feel we had on the previous website into this one as well., because for me this is somewhat missing at the moment. I know it's only a first draft and this is something I'd be happy to help with by the way! 😄

**Pull request merged**